tests/serialization/test_dag_serialization.py

```diff
@@ -82,6 +82,7 @@
                 "ui_color": "#f0ede4",
                 "ui_fgcolor": "#000",
                 "template_fields": ['bash_command', 'env'],
+                "template_fields_renderers": {},
```

Comment:
```suggestion
                "template_fields_renderers": {'bash_command': 'bash'},
```
WDYT?

Reply:
Well, this will require change in BashOperator but I will do

Reply:
Ah sorry I somehow thought that is not the default implementation.

Reply:
Maybe we should add template field renderers for the basic / core operators here?

Or in another PR. I am fine with either.

Reply:
> Or in another PR.

That's exactly what I planned as this change does not affect them